packages/neuvector/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,58 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -4,8 +4,8 @@
+ 
+ openshift: false
+ 
+-registry: registry.neuvector.com
+-tag: 4.4.3
++registry: docker.io
++tag: 5.0.0-preview.1
+ oem:
+ imagePullSecrets:
+ psp: false
+@@ -20,7 +20,7 @@
+       maxSurge: 1
+       maxUnavailable: 0
+   image:
+-    repository: neuvector/controller
++    repository: neuvector/controller.preview
+   replicas: 3
+   disruptionbudget: 0
+   schedulerName:
+@@ -154,7 +154,7 @@
+   # If false, enforcer will not be installed
+   enabled: true
+   image:
+-    repository: neuvector/enforcer
++    repository: neuvector/enforcer.preview
```

Comment:
All of these also need to be either in rancher dockerhub or mirrored to rancher dockerhub. I believe we said the latter, so we need to make those repo's and get them mirrored.